MODULE.bazel

```diff
@@ -9,9 +9,12 @@ bazel_dep(name = "platforms", version = "0.0.7")
 deps = use_extension("//internal:extensions.bzl", "shellcheck_dependencies")
 use_repo(
     deps,
-    "shellcheck_darwin_amd64",
-    "shellcheck_linux_amd64",
+    "shellcheck_darwin_aarch64",
+    "shellcheck_darwin_x86_64",
+    "shellcheck_linux_x86_64",
     "shellcheck_linux_arm64",
```

Comment:
The new repo does not have arm64 for linux and it has to be aarch64.

Reply:
Good catch. This has been fixed, and running `bazel fetch //...` now succeeds.